README.md

```diff
@@ -61,10 +61,10 @@ Import:
 
 ```js
 // ESM
-import { detectPackageManager, addDependency } from 'nypm'
+import { detectPackageManager, addDependency, addDevDependency, addDevDependency, removeDevDependency } from 'nypm'
```

Comment:
There is no need for an `addDevDependency` function, `addDependency` has an `dev` option.

And also you don't need a `removeDevDependency` function, the package managers removes/uninstalls dependency with their commands, wwithout checking if it is a devDependency or not, it will be removed.

Reply:
This was from the linked issue's idea to introduce shortcuts.

Reply:
Okay, but actually you dont need a removeDevDependency function, the package managers removes/uninstalls dependency with their commands, wwithout checking if it is a devDependency or not, it will be removed.